backpack-common/src/main/res/values/colors.xml

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+  <color name="bpkTextPrimary">#000000</color>
```

Comment:
this is... annoying. i found that when running the compose tests that there was a compilation error. we refer to the primary colour in the icons. in compose tinting will happen elsewhere. in the old view system this will get overridden in the other module